content/bash.rst

```diff
@@ -1,7 +1,7 @@
 Bash
 ####
 
-:menu_order: 155
+:menu_order: 015
```

Comment:
this is a number use to sort the elements in the menu... so you can just use `15` :)